### ISSUE TITLE
Can't overwrite default config URLs

### ISSUE THREAD

**Issue by viniciusbig** [CakeDC/users#902]
TL;DR

It is not possible to use the config file (users.php) to customize and change the defaults URLs configurations.

The config file is loaded before setupConfigUrls(), so its not possible to customize it.

```php
// vendor/cakedc/users/config/bootstrap.php
Configure::load('CakeDC/Users.users');
collection((array)Configure::read('Users.config'))->each(function ($file) {
    Configure::load($file);
});
UsersUrl::setupConfigUrls();  // too late to customize URLs here
```

---

### Details

I'm trying to implement a new endpoint to authenticate (`/api/users/token`), similar to the default `/login` but to return a JWT token. Since `username` and `password` are sent trough POST, the `CakeDC\Auth\Authenticator\FormAuthenticator` is used to handle it.

This class then uses `Authentication\Authenticator\FormAuthenticator` to authenticate but before doing that, it checks the request URL against the `Auth.Authenticators.Form.loginUrl` configuration.

`CakeDC/Users` sets the `Auth.Authenticators.Form.loginUrl` in plugin's bootstrap. But the URL config runs after the load of the config fil. See vendor/cakedc/users/config/bootstrap.php line 21.

Probably the `UsersUrl::setupConfigUrls()` line should be called before the `Configure::load()`

**Comment by rochamarcelo** [comment]
`UsersUrl::setupConfigUrls()` rely on user config, so it should be called after `Configure::load`. It seems that you need to manually bootstrap the plugin (copying if needed infos you from plugin bootstrap.php )

**Comment by viniciusbig** [comment]
You are right, @rochamarcelo, thanks!

I was adding the wrong config key in the users.php. Now I added the correct one.

For future references:

```php
config/users.php

//...
'Auth' => [
    'Authenticators' => [
        'Form' => [
            'loginUrl' => [
                [
                    'prefix' => false,
                    'plugin' => false,
                    'controller' => 'Users',
                    'action' => 'login',
                ],
                [
                    'prefix' => 'api',
                    'plugin' => false,
                    'controller' => 'Users',
                    'action' => 'token',
                ]
            ]
        ],
    ],
],
```

**Comment by rochamarcelo** [comment]
Thank you.
